frontend/src/components/pages/acls/PrincipalGroupEditor.tsx

```diff
@@ -311,24 +311,9 @@ const ResourceACLsEditor = observer((p: {
                 </Label>
             )}
 
-            <Label text="Operations">
-                <div
-                    style={{
-                        display: 'flex',
-                        flexDirection: 'column',
-                        rowGap: '8px',
-                        columnGap: '20px',
-                        flexWrap: 'wrap',
-                        width: '100%',
-                        maxHeight: '60px',
-                        alignItems: 'flex-start'
-                    }}
-                >
+            <Label text="Operations" style={{width: '100%'}}>
```

Comment:
Do we need the remaining styles from the diff?

Reply:
Unfortunately yes, but I plan to remove these `Operations` label completely in the future iteration